pages/profilepage.html

```diff
@@ -16,33 +16,48 @@
     rel="stylesheet"
     href="https://unpkg.com/swiper@7/swiper-bundle.min.css"
     />
+    <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
```

Comment:
We already have JQuery in this project. You can import it locally as seen in other files. No need connecting to an unsecured endpoint as to fetch it.